tests/unit/state-getters.spec.ts

```diff
@@ -0,0 +1,160 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { shallowMount } from '@vue/test-utils'
+import { getVuexStore } from '@/store'
+import { Actions } from '@/components/common'
+
+Vue.use(Vuetify)
+const vuetify = new Vuetify({})
+const store = getVuexStore()
+
+describe('State Getters', () => {
+  let vm: any
+
+  beforeAll(async () => {
+    // mount the component and wait for everything to stabilize
+    // (this can be any component since we are not really using it)
+    const wrapper = shallowMount(Actions, { store, vuetify })
+    vm = wrapper.vm
+    await Vue.nextTick()
+  })
+
+  it('returns correct values for "Is Busy Saving" getter', async () => {
+    // initially, these getters should be false
+    expect(vm.isSaving).toBe(false)
+    expect(vm.isSavingResuming).toBe(false)
+    expect(vm.isFilingPaying).toBe(false)
+    expect(vm.isBusySaving).toBe(false)
+
+    // verify that the Is Saving flag works
+    await vm.$store.commit('mutateIsSaving', true)
```

Comment:
NB: this is how you update the store after mounting the component. Note that it's async.